.circleci/config.yml

```diff
@@ -41,6 +93,12 @@ jobs:
               else
                 echo "Skipping coveralls"
               fi
+  validate-files:
+      docker:
+        - image: python:3.8
+      steps:
+        - checkout
+        - *restore_cache
```

Comment:
why the `*`

Reply:
It injects the alias - https://circleci.com/docs/2.0/writing-yaml/.